.github/maven.settings.xml

```diff
@@ -57,7 +65,7 @@
             <password>${env.NEXUS_RELEASE_PASSWORD}</password>
```

Comment:
`NEXUS_RELEASE_USERNAME` and `NEXUS_RELEASE_PASSWORD` variable do not exist on GitHub Actions